diff/12-dec-vs-29-jan.html

```diff
@@ -3,6 +3,12 @@
 ---
 
 <table class="table table-diff">
+<colgroup>
```

Comment:
this will get lost i imagine when we regenerate the diff. May need to look at the scripts/diff2html.py script ...